### PR TITLE
tests: improve unit test code coverage for SlideUpViewPresenter.swift (SDKCF-6383)

### PR DESCRIPTION
**Pull request by Esakkiraja-Pothikannan**
# Description
- Added slide-up-trigger-without-messageBody.json file with an invalid response
- Added coverage to guard else statement in viewDidInitialize()

## Links
SDKCF-6383

# Checklist
- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [x] I have added to the [changelog](../blob/master/CHANGELOG.md#Unreleased)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data **before every commit**, including API endpoints and keys, and internal links
- [x] I ran `fastlane ci` without errors
- [x] All project file changes are replicated in `SampleSPM/SampleSPM.xcodeproj` project

### CODE REVIEW

File: Tests/UITests/ResponseStubs/slide-up-close.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
                     "header": "Test Campaign",
                     "titleColor": "#000000",
                     "headerColor": "#000000",
-                    "messageBodyColor": "#000000",
-                    "backgroundColor": "#ffffff",
+                    "messageBodyColor": "",
```
Comment (imaginaris) [inline Tests/UITests/ResponseStubs/slide-up-close.json:28]: Why was this change needed?

**Review by imaginaris — APPROVED** [review]

File: Tests/UITests/ResponseStubs/slide-up-close.json (hunk 1 above)
Comment (Esakkiraja-Pothikannan) [reply Tests/UITests/ResponseStubs/slide-up-close.json:28]: These changes add coverage to nil coalescing of [this](https://github.com/rakutentech/ios-inappmessaging/blob/ad881ea56ad734e8562bccab91a968461fbf83db/Sources/RInAppMessaging/Views/Presenters/SlideUpViewPresenter.swift#L25-L29) lines.
